tools/eval.py

```diff
@@ -33,30 +36,26 @@ def main(cfg):
             parallel_mode="data_parallel",
             gradients_mean=True,
         )
+        set_logger(
+            name="mindocr", output_dir=cfg.train.ckpt_save_dir or "./", log_fn=f"log_eval_{rank_id}.txt", rank=rank_id
```

Comment:
The name of logger here (`mindocr`) is different form initialized above (`mindocr.eval`)